ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -11,8 +11,13 @@ MAINTAINER Dirk Thomas dthomas+buildfarm@@osrfoundation.org
 VOLUME ["/var/cache/apt/archives"]
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
+
+@(TEMPLATE(
+    'snippet/setup_locale.Dockerfile.em',
+    os_name=os_name,
+    os_code_name=os_code_name,
+    timezone=timezone if timezone else None,
```

Comment:
Have you tried to expand this template which doesn't pass in a `timezone` variable?